src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner.vue

```diff
@@ -1,80 +1,105 @@
 <template>
-	<div class="sitewide-appeal-wrapper">
-		<div class="sitewide-appeal row" @click="toggleAccordion">
-			<div class="sitewide-header small-12 medium-9 medium-offset-2 large-9 large-offset-2  columns">
-				<h2>
-					<span v-html="bannerHeadline" class="strong"></span>
-					<kv-icon
-						@click="toggleAccordion"
-						:class="{ flipped: open }"
-						class="toggle-arrow"
-						name="medium-chevron"
-						:from-sprite="true"
-					/>
-				</h2>
-			</div>
+	<div ref="pageOverlay">
+		<div class="sitewide-appeal-wrapper">
+			<div class="sitewide-appeal row" @click="toggleAccordion">
+				<div class="sitewide-header small-12 medium-9 medium-offset-2 large-9 large-offset-2  columns">
+					<h2>
+						<span v-html="bannerHeadline" class="strong"></span>
+						<kv-icon
+							@click="toggleAccordion"
```

Comment:
It would be good to wrap this icon in a `<button>` tag, and place the @click handler there so keyboards can focus it, and it will announce to screenreaders.

Reply:
I'd recommend moving the button+icon outside of the h2.

Reply:
Making this change resulted in unexpected css issues, since this is existing code and the banner stuff needs to get out I'm going to move forward. More than willing to talk with you about it Ryan.